azure-iot-hub-devicesdk/tests/data_structures/test_method_request.py

```diff
@@ -0,0 +1,22 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import pytest
+from azure.iot.hub.devicesdk.data_structures import MethodRequest
```

Comment:
>data_structures [](start = 29, length = 15)

ok you're gonna hate me for being nitpicky. "data structures" to me refers to "primary" data structures like arrays, lists, trees, etc.
I think what is contained in this package is types (I know this may sound egregious in python but it is what it is: a class definition creates a type).

Reply:
I discussed this with Bert, and we think a better name is actually "common" so that it doesn't restrict us to just types. The whole purpose of this package is to provide common code for use between internal packages.

---
In reply to: [267185961](https://github.com/Azure/azure-iot-sdk-python-preview/pull/61#discussion_r267185961) [](ancestors = 267185961)